android/capacitor/src/main/java/com/getcapacitor/BridgeWebChromeClient.java

```diff
@@ -134,8 +139,33 @@ public void onGeolocationPermissionsShowPrompt(String origin, GeolocationPermiss
   }
 
   @Override
-  public boolean onShowFileChooser(WebView webView, ValueCallback<Uri[]> filePathCallback, FileChooserParams fileChooserParams) {
-    return super.onShowFileChooser(webView, filePathCallback, fileChooserParams);
+  public boolean onShowFileChooser(WebView webView, final ValueCallback<Uri[]> filePathCallback, FileChooserParams fileChooserParams) {
+    Intent intent = fileChooserParams.createIntent();
+    if (fileChooserParams.getMode() == FileChooserParams.MODE_OPEN_MULTIPLE) {
+      intent.putExtra(Intent.EXTRA_ALLOW_MULTIPLE, true);
+    }
+    try {
+      bridge.cordovaInterface.startActivityForResult(new CordovaPlugin() {
```

Comment:
Why using the Cordova stuff for this? We should have some of our own utils

Reply:
I explained it on the PR comment.

The capacitor's Bridge startActivityForPluginWithResult needs a PluginCall and a Plugin to work, but this will be called from the input type file button click, so there won't be a real Plugin or PluginCall invoking it.
I could have tried faking a Plugin and PluginCall, but doing it with the fake CordovaPlugin was easier.

Reply:
Sounds good